packages/enet.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Enet < Package
+  description 'ENet reliable UDP networking library'
+  homepage 'https://github.com/lsalzman/enet'
+  git_hashtag '4f8e9bdc4ce6d1f61a6274b0e557065a38190952'
+  @_ver = git_hashtag[0, 7]
```

Comment:
No need to save `@_ver` if it does not require anymore

Reply:
I'm not sure what you mean.